pesacheck_meedan_bridge/.env.example

```diff
@@ -0,0 +1,10 @@
+PESACHECK_URL=https://pesacheck.org/feed
+PESACHECK_RSS2JSON_API_KEY=
+PESACHECK_CHECK_URL=https://check-api.checkmedia.org/api/graphql
+PESACHECK_CHECK_TOKEN=
+PESACHECK_CHECK_WORKSPACE_SLUG=pesacheck-tipline-sandbox
+PESACHECK_SENTRY_DSN=
+PESACHECK_SENTRY_TRACES_SAMPLE_RATE=1.0
+PESACHECK_SENTRY_ENVIRONMENT=local
+# Only for Dokku to allow app to be deployed from docker image
+DOKKU_DOCKERFILE_START_CMD="tail -f /dev/null"
```

Comment:
Why isn't this part of Dockerfile itself then?

Reply:
@kilemensi This command is only needed to successfully deploy to Dokku.
If we add it to Dockerfile, it will work as expected in Dokku but creates a problem with `pants run pesacheck_meedan_bridge/docker:pesacheck_meedan_bridge`
The process does not exit after docker build completion even with `Ctrl + C` that is why I restricted it to Dokku env.

Reply:
... not sure I follow @koechkevin . I see this similar thing on [`docker-compose.yml`](https://github.com/CodeForAfrica/api/blob/0637ba70d09795662d1ed6d6c43c2d1e79608ac2/docker-compose.yml#L53) as well. It seems like we need it more than just for Dokku.

Either way, why would pants have this issue? Do you get the same issue if you run the image using standard docker/docker compose?

Reply:
No, It is only with pants @kilemensi

Reply:
Then I think lets put it in Dockerfile. We'll debug pants later (as long as docker compose works).